lib/mayaUsd/ufe/UsdUIInfoHandler.h

```diff
@@ -47,6 +50,17 @@ class MAYAUSD_CORE_PUBLIC UsdUIInfoHandler : public Ufe::UIInfoHandler
     Ufe::UIInfoHandler::Icon treeViewIcon(const Ufe::SceneItem::Ptr& item) const override;
     std::string              treeViewTooltip(const Ufe::SceneItem::Ptr& item) const override;
     std::string              getLongRunTimeLabel() const override;
+
+private:
+    void updateInvisibleColor();
+
+    // Note: the on-color-changed callback function is declared taking a void pointer
+    //       to be compatible with MMessage callback API.
+    static void onColorChanged(void*);
+
+    std::array<double, 3> fInvisibleColor;
+    bool                  fInvisibleColorValid = false;
```

Comment:
Aren't the RGB values between 0 and 1? So could we init this array to -1 values and use that as a valid check instead of using an extra boolean?